src/config.rs

```diff
@@ -82,6 +82,50 @@ macro_rules! create_config {
             $(pub $i: Option<$ty>),+
         }
 
+        // This trait and the following impl blocks are there only so that we
+        // can use UCFS inside the get_docs() function on builtin types for configs.
+        trait IsConfigType {
+            fn get_variant_names() -> Vec<&'static str>;
```

Comment:
I think I understand; you mean as opposed to returning a `Vec<'static str>`, combine in `get_variant_names()` and only return `&'static str`?